.travis.yml

```diff
@@ -48,7 +48,7 @@ install:
       if [[ -f "$RECIPE_OUTPUT" && "$DO_UPLOAD" = "true" ]]; then anaconda -t $ANACONDA_TOKEN upload "$RECIPE_OUTPUT" -u quantopian --label ci; fi
     done
 
-  - conda create -n testenv --use-local --yes -c quantopian pip python=$TRAVIS_PYTHON_VERSION numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION libgfortran=3.0 ta-lib=$TALIB_VERSION
+  - conda create -n testenv --use-local --yes -c quantopian pip python=$TRAVIS_PYTHON_VERSION numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION libgfortran=3.0 ta-lib=$TALIB_VERSION mkl=11.3.3-0
```

Comment:
It doesn't seem like pegging mkl here actually helps with the current travis problems, so I'd probably just remove this to avoid cluttering the git history.

Reply:
removed, bumped conda version to be 4.3.31, and added `--debug`